test/integ/quote-classic.test.ts

```diff
@@ -1695,65 +1683,302 @@ describe('quote', function () {
             const sendPortionEnabledValues = [true, undefined];
             GREENLIST_TOKEN_PAIRS.forEach(([tokenIn, tokenOut]) => {
               sendPortionEnabledValues.forEach((sendPortionEnabled) => {
-                it(`${tokenIn.symbol} -> ${tokenOut.symbol} sendPortionEnabled = ${sendPortionEnabled}`, async () => {
-                  // Arrange:
-                  // - token amount from swapper
-                  // - greenlist token pairs
-                  // - parametrize on sendPortionEnabled
-                  // - send deadline, recipient and simulateFromAddress to invoke tenderly simulation
-                  getAmount(1, type, tokenIn.symbol!, tokenOut.symbol!, '100');
-
-                  // Act:
-                  // - call classic quote
-
-                  // Assert:
-                  // - check if the response is 200
-                  // - check if the response contains the methodParameters
-                  // - check if the response contains portion-related payloads if sendPortionEnabled = true
-                  // - check if the response contains only portionBips and portionAmount if sendPortionEnabled not send, explicitly check for portionRecipient = undefined
-                  // - check if the response contains the simulationError
-
-                  // Act:
-                  // use the methodParameters to execute the swap
-
-                  // Assert:
-                  // - check if the swap is successful
-                  // - check if the token balances are correct
-                  //   - token balances assertion will stay the same for exact in and exact out
-                  //   - need to explicitly check if the portion recipient balances increases for exact in and exact out (new setup)
-                  //   - explicitly check that the portion recipient balances increase is the portion Bips against token out balance changes
-                });
+                const shouldSkip =
+                  // there's a known bug of portion service not supporting the native address 0x00.00 lookup
+                  // that will cause the native token portion tests to fail
+                  ((tokenIn.isNative || tokenOut.isNative) && sendPortionEnabled) ||
```

Comment:
we should definitely do a native test, you can just cast the zero address to whatever portion service can support now in the test case and remove it later